### PR TITLE
Update event-error-codes.md

### PR DESCRIPTION
**Pull request by msbemba**
Added event ID 54 to the document

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8996

File: microsoft-365/security/defender-endpoint/event-error-codes.md [hunk 1]
```diff
@@ -102,6 +102,8 @@ For example, if devices aren't appearing in the **Devices list**, you might need
...
    |52|New cloud configuration failed to apply, version: %1. Also failed to apply last known good configuration, version %2. Successfully applied the default configuration.|Received a bad configuration file from the cloud service. Failed to apply the last known good configuration - and the default configuration was applied.|The service will attempt to download a new configuration file within 5 minutes. If you don't see event #50 - contact Support.|
    |53|Cloud configuration loaded from persistent storage, version: %1.|The configuration was loaded from persistent storage on service startup.|Normal operating notification; no action required.|
+   |54|Global (per-pattern) state changed. State: %1, pattern: %2
+   If state = 0 Cyber-data reporting rule has reached its defined capping quota and won't send more data until the capping quota will expire. If state = 1: The capping quota expired and the rule will resume sending data.|Normal operating notification; no action required.|
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/event-error-codes.md]:
```suggestion
   If state = 0: Cyber-data reporting rule has reached its defined capping quota and won't send more data until the capping quota expires. If state = 1: The capping quota expired and the rule will resume sending data.|Normal operating notification; no action required.|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PehDeh** [comment]
will it store the events happening and send them once the capping quota is set back to 1 or drop them?

**Comment by msbemba** [comment]
> will it store the events happening and send them once the capping quota is set back to 1 or drop them?

It will store the events and resume sending them once the capping quota is set back to 1.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Pull request merged**